ProcessLib/SourceTerms/Python/PythonSourceTerm.h

```diff
@@ -0,0 +1,77 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include "NumLib/DOF/LocalToGlobalIndexMap.h"
+#include "NumLib/IndexValueVector.h"
```

Comment:
Looks like this particular one is not needed. Can you try to put some of the includes into cpp and use fwd decls here?
These should be possible to put as fwd decls:
PythonSourceTermPythonSideInterface
PythonSourceTermLocalAssemblerInterface.h
LocalToGlobalIndexMap